tests/Fixtures/TestBundle/Entity/DummyTableInheritanceRelated.php

```diff
@@ -78,4 +78,29 @@ public function setChildren($children)
 
         return $this;
     }
+
+    /**
+     * @param $child
+     *
+     * @return $this
+     */
+    public function addChild($child)
```

Comment:
the adder and remover were missing and dur to the same bug where shown in the POST response but never really persisted, now that the state is refresh, it needed to be persisted and thus also fixed this false positive test result